content/integrations/awsrds.md

```diff
@@ -66,6 +66,24 @@ If you haven't already, set up the [Amazon Web Services integration first](/inte
   16. Choose the **Enable Now** radio button.
   17. Click the **Create Function** button.
 
+<div class="alert alert-warning">
```

Comment:
Do we really need a warning here? Pointing out something that the user should just ignore doesn't seem warn-worthy.